.ci/test-project/spack_configs/darwin/spack.yaml

```diff
@@ -0,0 +1,18 @@
+# Spack env file for mac
+spack:
+  packages:
```

Comment:
I assume not listing any compilers here allows spack to find something?

Reply:
in the past we have had to list macOS compilers, but it could have be necessary b/c we were overriding defaults.

Reply:
I figured that was true. I assume Spack just goes with the system gnu. Maybe its better to just leave it blank and let Spack handle it.

Reply:
Added the comment 👍